Tests/DataPipeline/Core/UnitTest.swift

```diff
@@ -75,6 +75,7 @@ open class UnitTest: SharedTests.UnitTestBase<CustomerIO> {
 
     override open func cleanupTestEnvironment() {
         super.cleanupTestEnvironment()
+        Analytics.removeActiveWriteKey(dataPipelineConfigOptions.cdpApiKey)
```

Comment:
curious if we are removing `active write` key, why do we still need to set a random on each test case?

Reply:
Mainly for api tests as we are not actually removing the provided write keys for them and they may have multiple initializations as they are for only validating public apis.